runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -587,6 +587,15 @@ void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::Tr
    }
 
 #if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+static void anchorAllChildren_T(TR::Node * node, TR::TreeTop *anchorTree, TR::Compilation* comp)
```

Comment:
What does `anchorAllChildren_T` do differently compared to `OMR::Optimization::anchorAllChildren` to necessitate this? If the issue you were trying to solve is trying to access `OMR::Optimization::anchorAllChildren` from a static method, you could either pass `this` pointer or turn `insertVectorizedMisMatchArgumentChecksAndAdjustForOffHeap` into a member function of `RecognizedCallTransformer`.